addons/registry/2.7.1/install.sh

```diff
@@ -78,8 +78,10 @@ function registry_docker_ca() {
         bail "Docker registry address required"
     fi
 
-    mkdir -p /etc/docker/certs.d/$DOCKER_REGISTRY_IP
-    ln -s --force /etc/kubernetes/pki/ca.crt /etc/docker/certs.d/$DOCKER_REGISTRY_IP/ca.crt
+    if [ -n "$DOCKER_VERSION" ]; then
```

Comment:
would `DOCKER_REGISTRY_IP` be a better check here?

Reply:
That variable is set when using the registry with containerd too. I think `DOCKER_VERSION` is what we want here since that will only be set when using docker for the container runtime.